gruntfile.js

```diff
@@ -237,7 +237,27 @@ module.exports = function(grunt) {
           src: ['**'],
           dest: 'tmp/samples'
         }]
-      }
+      },
+      chrome_agent: {
+        files: [{
+          expand: true,
+          cwd: 'chrome_agent',
+          src: ['**'],
```

Comment:
Are you sure there is no extra files in there we don't want to copy (i don't know, i am asking :))